packages/services/f-braze-adapter/src/services/utils/transformCardData.js

```diff
@@ -49,6 +49,7 @@ const transformCardData = card => {
         restaurant_logo_url: restaurantLogoUrl,
         icon_1: icon = restaurantLogoUrl,
         location,
+        locations,
```

Comment:
Based on the ticket, we're not expecting `location` as a key within the extras, instead `location` can potentially have more than one location in, in a comma-separated fashion.